src/chrome/content/rules/ENS.fr.xml

```diff
@@ -167,8 +187,8 @@
 	<securecookie host="^(?:www-mail\.(?:biologie|magbio)|www\.cof|crypto(?:bib)?|\.?halley|\.mail\.(?:jourdan|lumiere)|www\.gdr-meso\.phys)\.ens\.fr$" name=".+" />
 
 
-	<rule from="^http://(dg|ecocampus)\.ens\.fr/"
-		to="https://www.$1.ens.fr/" />
+	<rule from="^http://dg\.ens\.fr/"
```

Comment:
https://github.com/Bisaloo/https-everywhere/blob/873973e277908d928a9b969ea519aca6eb6a4d89/src/chrome/content/rules/ENS.fr.xml#L71

This comment may seem wrong at the moment because `dg.ens.fr` is unreachable over both `http` and `https` but it is not usually the case. I am assuming this is just a temporary issue and it will be back to "normal" (invalid certificate) soon.